packages/babel-plugin-transform-es2015-arrow-functions/test/fixtures/arrow-functions/spec/expected.js

```diff
@@ -14,7 +14,7 @@ function foo() {
 
     return function () {
       babelHelpers.newArrowCheck(this, _this2);
-      return this;
```

Comment:
I'd prefer if we could keep `this` as `this`, but it's not like it's incorrect code here. Minifies better too!